app/src/main/java/com/android/calculator2/Calculator.kt

```diff
@@ -384,7 +387,17 @@ class Calculator : AppCompatActivity(), OnTextSizeChangeListener, EvaluateCallba
         // Play audio for numbers and operators
 
         if ((view.tag != null) && PlayerUtil.RAW_FILE_EQUALS != view.tag) {
-            ttsViewModel.speak(view.tag.toString().tagToSpokenText(), QueueMode.FLUSH, Random.nextInt().toString())
+            val spokenText = view.tag.toString().tagToSpokenText()
+            ttsViewModel.speak(spokenText, QueueMode.FLUSH, Random.nextInt().toString())
+
+            if (spokenText.isDigitsOnly()) {
+                LearningEventUtil.reportNumberLearningEvent(
+                    numberGson = NumberGson().apply {
+                        value = spokenText.toInt()
+                        symbol = spokenText },
```

Comment:
```suggestion
```

@tuancoltech Symbol is not used in the this app for any of our supported languages, as far as I know?

See examples of number symbols here:
- http://hin.elimu.ai/content/number/list
- http://tha.elimu.ai/content/number/list